src/platforms/apple/common/configuration/watchdog-terminations.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Watchdog Terminations
+sidebar_order: 10
+description: "Learn how to turn off Watchdog Termination tracking"
+---
+
+<Note>
+This integration is available since 8.0.0. It was named <PlatformLink to="/configuration/out-of-memory/">Out of Memory</PlatformLink> before the 8.0.0 release.
+</Note>
+
+This integration tracks <a href="https://developer.apple.com/documentation/xcode/addressing-watchdog-terminations">watchdog terminations</a> based on heuristics. This feature is available for iOS, tvOS, and Mac Catalyst, works only if the application was in the foreground, and doesn't track watchdog terminations for unit tests.
+
+When a typical unhandled error occurs, the Apple SDK writes a report to disk with the current state of your application, with details like the stack trace, tags, breadcrumbs, and so on, before the app terminates. When the watchdog terminates your app this happens without further notice, which means the SDK can't write a report to disk. A common reason the watchdog can terminate your app is an Out Of Memory problem. In the case the app is terminated because it hangs we don't create a watchdog termination event, instead an AppHangs event is created.
```

Comment:
```suggestion
When a typical unhandled error occurs, the Apple SDK writes a report to disk with the current state of your application, with details like the stack trace, tags, breadcrumbs, and so on, before the app terminates. When the watchdog terminates your app this happens without further notice, which means the SDK can't write a report to disk. A common reason the watchdog can terminate your app is an out of memory problem. If the app is terminated because it hangs, we don't create a watchdog termination event, but instead an `AppHangs` event is created.
```

Reply:
We call this `Out Of Memory` everywhere else, like in the old docs. I think we should keep that.